Resources/views/CRUD/base_edit_form_macro.html.twig

```diff
@@ -17,7 +17,9 @@
                             <p>{{ form_group.description|raw }}</p>
                         {% endif %}
 
-                        {% for field_name in form_group.fields if admin.formfielddescriptions[field_name] is defined %}
+                        {% for field_name in form_group.fields
+                            if (admin.formfielddescriptions[field_name] is defined
```

Comment:
I think yes:) 

What is more important is what form was prepared rather than what fields were set in the configuration.